### ISSUE TITLE
Need URI filters for scrobbles

### ISSUE THREAD

**Issue by fatg3erman** [mopidy/mopidy-scrobbler#28]
Scrobbling from Spotify to Last.FM is now done by a web 'app' - you simply enable scrobbling on your Spotify account and all tracks you play on Spotify on any device are automatically scrobbled. 

This creates an issue if you're using Mopidy-Scrobbler in that all Spotify tracks  played through Mopidy now get scrobbled twice - once by Mopidy-Scrobbler and once by Spotify. 

We need a list of uri schemes to NOT scrobble so that people can enable Spotify web scrobbling but still scrobble local tracks using Mopidy-Scrobbler.

**Comment by jjok** [comment]
Yeah, I was thinking that we could do with a config setting to either whitelist or blacklist each backend. eg I don't really want to scrobble from Mopidy-Podcast, but I do from Mopidy-Spotify and Mopidy-Gmusic.

I haven't done it yet though.
